features/admin.claims.v1/components/edit/local-claim/edit-additional-properties-local-claims.tsx

```diff
@@ -79,9 +85,43 @@ export const EditAdditionalPropertiesLocalClaims:
             [ featureConfig, allowedScopes ]
         );
 
+        const RESTRICTED_PROPERTY_KEYS: string[] = [ "isUnique" ];
+
+        const [ showWarning, setShowWarning ] = useState<boolean>(false);
```

Comment:
Keep the state variables together. Consider moving this below line 67.

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/7106/commits/a5fb2e912007474c3ca67b3f4466fb16eb19b772